portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -3658,6 +3658,55 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPS-126479."
+	@priority = "4"
+	test SubmitFormWhenSessionExpires {
+		property portal.acceptance = "false";
+		property portal.upstream = "true";
+		property test.name.skip.portal.instance = "Forms#SubmitFormWhenSessionExpires";
+		property web.xml.timeout = "2";
```

Comment:
@natocesarrego , we still need to run this test to check if the timeout property is working.

Reply:
Indeed, this is not working. I've also run other Poshi test with the same property and it's failed. :disappointed: 

Therefore, I'll remove the Poshi test from this pull and forward it.

Thanks